.circleci/config.yml

```diff
@@ -5,9 +5,9 @@ common-steps:
     run:
       name: Install system-level and Python dependencies
       command: |
-        apt-get update && apt-get install -y --no-install-recommends make python3 python3-venv python3-setuptools
-        python3 -m venv .venv
-        ./.venv/bin/pip install --require-hashes -r requirements/requirements.txt
+        apt-get update && apt-get install -y --no-install-recommends make python3 python3-pip
+        pip install poetry
```

Comment:
I've opted here intentionally for an unconstrained install strategy since this repo probably is a good place to notice behavior changes in `poetry` early, but for other use cases we likely want to be more conservative.

Note that Poetry's install docs recommend `curl`-ing a script straight from https://install.python-poetry.org/ and running it, which seems problematic even in a container.

Reply:
I think this is fine until we switch to bookworm which will have poetry as a package. As for pinning, the poetry docs (https://python-poetry.org/docs/#ci-recommendations) say 
>Whatever method you use, it is highly recommended to explicitly control the version of Poetry used, so that you are able to upgrade after performing your own validation.

Reply:
I've added pinning now. If this repo survives (it might end up being merged into `securedrops-docs`) I think it's a good candidate to serve as a low-risk canary for new releases, but that could be an explicit decision for whichever repos we designate for such purposes.